3-extra/1-count-words.js

```diff
@@ -24,9 +24,19 @@
 */
 
 function countWords(string) {
-  const wordCount = {};
-
-  // write code here
+  let wordCount = {};
+  let arr = [];
+  if (string.length > 0) {
+    arr = string.split(" ");
+  }
```

Comment:
What is the output if we input `countWords(" ")`?

Reply:
If would like { '': 1 } instead of { '': 2 }, we can insert trim().arr = string.trim().split(" ");